src/D2L.Bmx/Okta/OktaClient.cs

```diff
@@ -0,0 +1,89 @@
+using System.Text.RegularExpressions;
+using System.Xml;
+using D2L.Bmx.Okta.Models;
+using D2L.Bmx.Okta.State;
+namespace D2L.Bmx.Okta;
+
+internal interface IOktaClient {
+	public string Name { get; }
```

Comment:
do we still need this at all? @boarnoah 
```suggestion
```

Reply:
No, that was another attribute of me trying to over-generalize the Okta provider bit